tests/io_utils.go

```diff
@@ -117,11 +120,50 @@ func FixErrorDevice(nodeName string) {
 
 }
 
+func executeDeviceMapperOnNode(nodeName string, cmd []string) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	Expect(err).ToNot(HaveOccurred())
+
+	// Image that happens to have dmsetup
+	image := fmt.Sprintf("%s/virt-launcher:%s", flags.KubeVirtRepoPrefix, flags.KubeVirtVersionTag)
+	pod := &k8sv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "device-mapper-pod-",
+		},
+		Spec: k8sv1.PodSpec{
+			RestartPolicy: k8sv1.RestartPolicyNever,
+			Containers: []k8sv1.Container{
+				{
+					Name:    "launcher",
+					Image:   image,
+					Command: cmd,
+					SecurityContext: &k8sv1.SecurityContext{
+						Privileged: pointer.BoolPtr(true),
+						RunAsUser:  pointer.Int64Ptr(0),
+					},
+				},
+			},
+			NodeSelector: map[string]string{
+				"kubernetes.io/hostname": nodeName,
+			},
+		},
+	}
+	pod, err = virtClient.CoreV1().Pods(util.NamespaceTestDefault).Create(context.Background(), pod, metav1.CreateOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	Eventually(func() bool {
```

Comment:
You could use

```
Eventually(ThisPod(pod), 30).Should(HaveSucceeded)
```

We have some more or less new useful matchers now: https://github.com/kubevirt/kubevirt/tree/main/tests/framework/matcher

Reply:
Done!